kafka-config.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
generell praksis i nav er å ha IAC repos for dette

Reply:
Flyttes til secrets

Reply:
har endret kode slik at restbrukernavn og kafkaclientid hentes fra Vault. Default verdier som settes i kode, brukes for lokal testing..

Reply:
Jeg fant eksempel på commit av json fil med kafka roller i applikasjonen sykepenger-spleis. Har egentlig litt vanskelig for å se at det er stor sikkerhetsrisiko med å legge ut brukernavnet så lenge passordet er skjult og kafka køen er i fss. Dessuten er det med minimal kunnskap om NAV sin navnepraksis, lett å gjette seg til brukernavn når man har applikasjonsnavn. Men erstatter brukernavnene på produsent/konsument med TODO.